WooCommerce/Classes/POS/Analytics/PointOfSaleAnalytics.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import protocol WooFoundation.Analytics
+import protocol WooFoundation.AnalyticsProvider
+
+final class PointOfSaleAnalytics: Analytics {
+    var userHasOptedIn: Bool {
+        get {
+            guard let _: Bool? = UserDefaults.standard.object(forKey: .userOptedInAnalytics) else {
+                return false
+            }
+            return true
+        }
+        set {
+            UserDefaults.standard.set(newValue, forKey: .userOptedInAnalytics)
+        }
```

Comment:
For now I'm using the default settings that a merchant has already stored via the WooCommerce app. For MVP we may want to handle this separately.